packages/fast-components-react-msft/package.json

```diff
@@ -84,6 +84,7 @@
     "babel-preset-react": "^6.24.1",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
+    "eyes.images": "0.0.52",
```

Comment:
does this get used?

Reply:
Yes, but, on next PR. I can remove for now.